src/components/search-bar/search-bar.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from "react";
+import searchIcon from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/basic/icon-search.svg";
+
+export interface SearchBarProps {
+  q: string;
+  setQuery: React.Dispatch<React.SetStateAction<string>>;
+  searchHeaderUrl?: string;
+  altText?: string;
+  inputPlaceholder?: string;
+}
+
+const SearchBar: React.FC<SearchBarProps> = ({
+  q,
+  setQuery,
+  searchHeaderUrl = "/search",
+  altText = "search icon",
+  inputPlaceholder = "Search here"
```

Comment:
Just a note: All texts are depending on how we want to implement strings in the future. But let's keep it like this for now.

Reply:
True, let us see how we solve the implementation for system texts in the future.